python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -757,6 +765,11 @@ def runPipeline(
                             # Do not save fields that are not set.
                             out.write(report.json(exclude_none=True, indent=2))
 
+        if args.coverage:
+            cov.stop()
+            cov.html_report(directory="covhtml")
```

Comment:
A textual summary sent to stdout would be useful as well. It looks like `cov.report()` can do that.